elephant/test/test_statistics.py

```diff
@@ -750,7 +752,7 @@ def test_instantaneous_rate_regression_245(self):
         # This test makes sure that the correct kernel width is chosen when
         # selecting 'auto' as kernel
         spiketrain = neo.SpikeTrain(
-            range(1, 30) * pq.ms, t_start=0 * pq.ms, t_stop=30 * pq.ms)
+            pq.ms * range(1, 30), t_start=0 * pq.ms, t_stop=30 * pq.ms)
```

Comment:
quantities works with `range()`, not the other way, but this could be changed back for better readability